aif360/metrics/binary_label_dataset_metric.py

```diff
@@ -127,7 +127,7 @@ def consistency(self, n_neighbors=5):
 
         .. math::
            1 - \frac{1}{n\cdot\text{n_neighbors}}\sum_{i=1}^n |\hat{y}_i -
```

Comment:
This line is what I was referring to. It has an extra n_neighbors now that should be removed.

Also, if you could put the `n_neighbors` in \text mode that will make it render nicer.

Reply:
Ah yes, okay. Done, hopefully it works. In LaTeX you need to escape underscores but not here it seems.